rails/lib/materials/data_helpers.rb

```diff
@@ -275,7 +275,7 @@ def materials_data( materials,
           archive_url:   (material.is_a?(ExternalActivity) && policy(material).archive?) ? view_context.archive_external_activity_url(material) : nil,
           copy_url: external_copyable(material) ? view_context.copy_external_activity_url(material) : nil,
           assign_to_class_url: current_visitor.portal_teacher && material.respond_to?(:offerings) ? "javascript:PortalComponents.renderAssignToClassModal({material_id: #{material.id}, material_type: '#{material.class.to_s}', lightbox_material_text: '#{t('material').pluralize.capitalize}', skip_reload: true})" : nil,
-          assign_to_collection_url: current_visitor.has_role?('admin') && material.respond_to?(:materials_collections) ? "javascript:get_Assign_To_Collection_Popup(#{material.id},'#{material.class.to_s}')" : nil,
+          assign_to_collection_url: current_visitor.has_role?('admin') && material.respond_to?(:materials_collections) ? "/search/get_current_material_unassigned_collections/?material_id=#{material.id}&material_type=#{material.class.to_s}" : nil,
```

Comment:
If this was moved to the ExternalActivity controller then a route path helper can be used here:
`edit_collections_external_activity_path(@material)`